for-developers/notes-datafest-20201.md

```diff
@@ -1,37 +0,0 @@
-Notes for Datafest 2021 Workshop, 
```

Comment:
@kuriwaki, I wouldn't delete this.  It's tucked away in GitHub and not interfering related to R.  (Notice it's listed in .Rbuildignore: https://github.com/IQSS/dataverse-client-r/blob/d19b2410ed31867df5d8341bd7471bce6823ce15/.Rbuildignore#L3)

Or if you do delete it, replace it with something similar for new people to get a quick feel for the package.